plugins/woocommerce/tests/api-core-tests/tests/shipping/shipping-method.test.js

```diff
@@ -30,11 +35,91 @@ const methodCostIndex = 2;
  */
 test.describe('Shipping methods API tests', () => {
 
+	test('cannot create a shipping method', async ({
```

Comment:
Maybe we can consider adding in a comment here on why we're testing this case (e.g. the API isn't allowing us for `x` reason so we're testing that here), since looking at this test it seems like it _should_ work, so it may be good to highlight why we're testing this case.

Reply:
Fully agree @zhongruige - thanks!

Reply:
Thank you for fixing these!